velox/docs/functions/array.rst

```diff
@@ -163,3 +163,14 @@ Array Functions
     If the arguments have an uneven length, missing values are filled with ``NULL`` ::
 
     SELECT zip(ARRAY[1, 2], ARRAY['1b', null, '3b']); -- [ROW(1, '1b'), ROW(2, null), ROW(null, '3b')]
+
+.. function:: array_sort(array(E)) -> array(E)
```

Comment:
In the documentation, functions appear in alphabetical order. Would you move this function to preserve that order?

Reply:
Sure. Done